### ISSUE TITLE
Option to select durring instalation Serbian Latin or Serbian Cyrilic

### ISSUE THREAD

**Issue by quenode** [magento/magento2#12256]
Hi Magento community,

During installation when selecting default language for magento as
Serbian Language the default language store sr_Cyrl_RS "
general/locale/code - sr_Cyrl_RS" .
This ok but we should have two option Serbian@Latin and Serbian@Cyrilic

Here are the patches.

```
/zendframework1/library/Zend/Locale/Data/likelySubtags.xml.orig
898,899d897
<               <likelySubtag from="sr_latin" to="sr_Latn_RS"/>
<               <!--{ Serbian; ?; ? } => { Serbian; Latin; Serbia }-->
```

This the Config.php

```
magento/framework/Locale/Config.php.orig
90,91c90
<         'sr_Cyrl_RS', /*Serbian Cyrillic (Serbia)*/
<         'sr_Latn_RS', /*Serbian Latin (Serbia)*/
---
>         'sr_Cyrl_RS', /*Serbian (Serbia)*/
```

**Comment by magento-engcom-team** [comment]
@quenode, thank you for your report.
We've created internal ticket(s) MAGETWO-84293 to track progress on the issue.

**Comment by danijelkusic** [comment]
How to switch Cyrillic to Latin? I mean, later in frontend...
I also posted a new topic on the link: https://github.com/magento/magento2/issues/13263

Please help. Thank you

**Comment by jahicDario** [comment]
When I added mentioned patches, in dropdown menu I got two locales with same name but different code. Is there a another file where name should be changed for sr_Latn_RS to be for example Serbian Latn? 
![selection_014](https://user-images.githubusercontent.com/17848628/50155787-4aeb5a00-02cd-11e9-92fa-8f134e12624b.jpg)

**Comment by magento-engcom-team** [comment]
Hi @quenode. Thank you for your report. 
The issue has been fixed in magento/magento2#22293 by @Bartlomiejsz in 2.3-develop branch
Related commit(s):
 - [a4f6b3c753c5df332be3545a26519581463d89f9](https://github.com/magento/magento2/commit/a4f6b3c753c5df332be3545a26519581463d89f9)
 - [1e6a2991f9ab3feef3baf2e0afdd70894ac9e7e9](https://github.com/magento/magento2/commit/1e6a2991f9ab3feef3baf2e0afdd70894ac9e7e9)
 - [29411feaac256670b5787d61217983fbdd974a88](https://github.com/magento/magento2/commit/29411feaac256670b5787d61217983fbdd974a88)

The fix will be available with the upcoming 2.3.4 release.
